tests/testthat/test_test_description.R

```diff
@@ -104,3 +104,18 @@ test_that("test_description can extract testthat describe-it style test descript
   expect_equal(as.character(desc), "abc: does the thing: ... and this behavior: does another thing")
   expect_equal(attr(desc, "type"), "testthat")
 })
+
+test_that("test descriptions can be extracted from implicit generic calls", {
+  # expect that we have a test for a standardGeneric
+  expect_silent({
+    s4generic_test <- Find(
+      function(i) class(i[[1]][[1]]) == "standardGeneric",
+      attr(examplepkg_cov, "tests")
+    )
+
+    s4generic_test_desc <- test_description(s4generic_test)
+  })
+
+  expect_equal(attr(s4generic_test_desc, "type"), "call")
+  expect_equal(as.character(s4generic_test_desc), "S4 Generic Call: show(<myS4Example>)")
+})
```

Comment:
Here I test that we can reproduce a test trace that is a call to a standard generic, and confirm that it produces a reasonable test description.